base.py

```diff
@@ -82,6 +83,37 @@ def __hash__(self):
         return hash(self.name)
 
 
+class Hospital(Location):
+
+    def __init__(self, env, capacity, name='vgh', location_type='hospital', lat=None, lon=None, cont_prob=None,
+                 recovery_prob=0.8):
+        super().__init__(env, capacity, name, location_type, lat, lon, cont_prob)
+        self.ventilators_in_use = 0
+        self.total_ventilators = 10
+        self.recovery_prob = recovery_prob
+
+    @property
+    def vacancy(self):
+        return self.capacity - len(self.humans)
+
+    def ventilator_available(self):
+        return self.total_ventilators - self.ventilators_in_use
+
+    def admit(self, human):
+        if self.vacancy > 0:
+            self.humans.add(human)
+            human.location = self
+        else:
+            raise ValueError("hospital full")
```

Comment:
It might be better to check for a vacant hospital close to `Human.location` and then call `self.at` for that location.

Reply:
fixed

Reply:
_select_location checks for closest hospital